src/main/scala/ai/starlake/schema/model/Domain.scala

```diff
@@ -231,6 +234,7 @@ case class LoadDesc(version: Int, load: Domain)
 
   def asDot(includeAllAttrs: Boolean, fkTables: Set[String]): String = {
     tables
+      .sortBy(_.name)
```

Comment:
sortBy(_.getFinaleName) probably

Reply:
This has no importance regarding rendering I think. It was introduced here in order to have an determinist output for unit test assertion.